src/test/ui/pattern/bindings-after-at/borrowck-pat-ref-mut-and-ref.rs

```diff
@@ -0,0 +1,133 @@
+#![feature(bindings_after_at)]
+#![feature(slice_patterns)]
+
+enum Option<T> {
+    None,
+    Some(T),
+}
+
+fn main() {
+    match &mut Some(1) {
+        ref mut z @ &mut Some(ref a) => {
+        //~^ ERROR cannot borrow `z` as immutable because it is also borrowed as mutable
+        //~| ERROR cannot borrow `_` as immutable because it is also borrowed as mutable
```

Comment:
hmm. Should we open a bug (or link to a pre-existing one) for the use of `_` in this diagnostic? Seems confusing to me.

Reply:
Yep, filed https://github.com/rust-lang/rust/issues/67565.